sympy/geometry/ellipse.py

```diff
@@ -1320,6 +1313,15 @@ class Circle(Ellipse):
         When trying to construct circle from three collinear points.
         When trying to construct circle from incorrect parameters.
 
+    If the equation of a circle, whose circle object is needed, is
+    ax**2 + by**2 + gx + hy + c = 0, then the input has to be of
+    the following format:
+
+    Circle(equation = ax**2 + by**2 + gx + hy + c, x='x', y='y')
```

Comment:
I think the `x='x'` and `y='y'` is not required with new changes.

Reply:
@ArifAhmed1995  You are right, I should've seen that earlier. Corrected now.